hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DFSAdmin.java

```diff
@@ -433,7 +433,7 @@ static int run(DistributedFileSystem dfs, String[] argv, int idx) throws IOExcep
    */
   private static final String commonUsageSummary =
     "\t[-report [-live] [-dead] [-decommissioning] " +
-    "[-enteringmaintenance] [-inmaintenance]]\n" +
+      "[-enteringmaintenance] [-inmaintenance] [-slownodes]]\n" +
```

Comment:
The corresponding documentation needs to update when CLI commands are added/updated.

Reply:
In fact it would appear confusion to HDFS administrators. These subcommands are meant to filter the DNs in these states, and "slownodes" is not a defined DataNode state.

Reply:
Reg the command options, I believe filters can be ideally used for both: 1) state of DNs (decommissioning, dead, live etc) and 2) nature of DNs (slow outliers). Updated the doc, please review.
Thanks